extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -535,6 +537,85 @@ def from_dict(self, model):
         self._c_lw = copy.deepcopy(model['_c_lw'])
         self._c_l = copy.deepcopy(model['_c_l'])
 
+    def _make_json_serializable(self, model):
+        """Makes the StringClassifier dict JSON serializable by
+        converting numpy data types to Python data types.
+
+        Args:
+            model: A dict representing a StringClassifier.
+
+        Returns:
+            model: A dict representing a StringClassifier
+        """
+
+        conversion_list_of_list_properties = [
+            '_b_dl',
+            '_c_dl',
+            '_c_lw'
+        ]
+
+        conversion_list_properties = [
+            '_c_l'
+        ]
+
+        conversion_properties = [
+            '_l_dp'
+        ]
+
+        for classifier_property in conversion_list_of_list_properties:
```

Comment:
Ah, I see -- I missed that. Sorry!

That said, can int/float be simplified as well? https://stackoverflow.com/questions/9452775/converting-numpy-dtypes-to-native-python-types